cli/Cargo.toml

```diff
@@ -1,12 +1,12 @@
 [package]
 name = "reinfer-cli"
 version = "0.11.0"
-description = "Command line interface for the reinfer platform."
+description = "Command line interface for the Re:infer conversational data intelligence platform"
```

Comment:
```suggestion
description = "Command line interface for Re:infer, the conversational data intelligence platform"
```

Reply:
I think I was trying to say "for the Re:infer conversational data intelligence platform", not that we are THE one